SPECS/kernel/config_aarch64

```diff
@@ -8158,6 +8152,7 @@ CONFIG_INFINIBAND_BNXT_RE=m
 CONFIG_INFINIBAND_CXGB4=m
 CONFIG_INFINIBAND_EFA=m
 # CONFIG_INFINIBAND_ERDMA is not set
+# CONFIG_MANA_INFINIBAND is not set
```

Comment:
result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/e5f84fbfb236129c180c1499013a2dd1cbd634e9